multi-educator/src/Dscp/Educator/Web/Educator/Queries.hs

```diff
@@ -0,0 +1,164 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Dscp.Educator.Web.Educator.Queries
```

Comment:
Now this is something which _definitely_ should be copypasted, wtf

Reply:
Yes, I just haven't got around removing it yet.